### ISSUE TITLE
Creating custom field (/fields) and showing/using it in assets (/hardware)

### ISSUE THREAD

**Issue by passboltUser** [snipe/snipe-it#5402]
#### Expected Behavior (or desired behavior if a feature request)

Creating a custom field and showing/using it in /hardware (assets). 

-----

#### Actual Behavior

I go to `/fields` and create a custom field:
![1_snipe](https://user-images.githubusercontent.com/29798595/39056691-b1b722d4-44b7-11e8-8943-f87577e28d0c.png)
Then I go to `/import`, upload a .csv file, click the `Process` button, select that newly created custom field and click the `Import` button. So far so good. Now I go to `/hardware` and expect there to see the newly created custom field, but it's unfortunately not there:
![2_snipe](https://user-images.githubusercontent.com/29798595/39056770-dd706bce-44b7-11e8-95ec-828f1066bc08.png)

I'm not sure but maybe I'm using it wrongly?

-----

#### Please confirm you have done the following before posting your bug report:

- [ ] I have enabled debug mode 
- [✔] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

-----
#### Provide answers to these questions:

- Is this a fresh install or an upgrade? 
A fresh install.
- Version of Snipe-IT you're running
`4.2.0 - build 3497`
- Version of PHP you're running
`PHP 7.0.28-0ubuntu0.16.04.1`
- Version of MySQL/MariaDB you're running
`mysql  Ver 15.1 Distrib 10.0.34-MariaDB, for debian-linux-gnu (x86_64) using readline 5.2`
- What OS and web server you're running Snipe-IT on
`Ubuntu 16.04.4 LTS`
- What method you used to install Snipe-IT (install.sh, manual installation, docker, etc)
https://snipe-it.readme.io/v4.2.0/docs/installation
- WITH DEBUG TURNED ON, if you're getting an error in your browser, include that error
- What specific Snipe-IT page you're on, and what specific element you're interacting with to trigger the error
- If a stacktrace is provided in the error, include that too.
- Any errors that appear in your browser's error console.
- Confirm whether the error is reproducible on the demo: https://snipeitapp.com/demo.
Yes.
- Include any additional information you can find in `storage/logs` and your webserver's logs.
- Include what you've done so far in the installation, and if you got any error messages along the way.
- Indicate whether or not you've manually edited any data directly in the database

Please do not post an issue without answering the related questions above. If you have opened a different issue and already answered these questions, answer them again, once for every ticket. It will be next to impossible for us to help you.

https://snipe-it.readme.io/docs/getting-help

**Comment by EarlRamirez** [comment]
@passboltUser I was able to reproduce what you are seeing if I understand correctly, I created a custom field and went to /hardware and I did not see the newly created custom field; however, after it was added to a custom field set that is already assigned to a few models I was able to see the newly created custom field from the drop-down list under /hardware.

You can also check out **https://snipe-it.readme.io/docs/custom-fields**

**Comment by passboltUser** [comment]
Thanks, I went to `/models`, under the one asset model I clicked the Update button and under the Field set drop down menu, I choose my newly created custom field set. My additional fields appear under `/import` and `/hardware` now. Unfortunately, when I try to `/import`, although the fields are shown/mapped currectly and the import succeeds, under `/hardware` my newly created fields are empty.

**Comment by EarlRamirez** [comment]
Is it the same model that you have added the custom field set too? You will see all the fields under the /hardware page; however, the model must have the custom field set that contains the fields that you want to be populated.

**Comment by passboltUser** [comment]
On the /hardware page I set the checkbox on the left and clicked Edit Go button on the top left, then under the 3rd field from top, Model, I clicked it and in the drop down menu I choose that only one notebook model, and clicked on the Save button. Now these 4 fields are filled. Thanks. My only question now is why do I had to do this manual selecting?

**Comment by EarlRamirez** [comment]
You don't need to modify each asset you can select the models and use **Bulk Edit** and add the custom field set like below.

![image](https://user-images.githubusercontent.com/12306859/39131752-368f862c-46de-11e8-9233-9d478bed7642.png)
and 
![image](https://user-images.githubusercontent.com/12306859/39131795-507f7d44-46de-11e8-98ad-e9d4fb2de04c.png)
Now all models will reflect these fields.

**Comment by passboltUser** [comment]
Thanks @EarlRamirez. I went to /models, but I only have 1 test model. I have many old entries and choose to use a dummy model. I went to /hardware, selected all the imported assets, clicked Edit and choose that one dummy model. It worked and my 4 custom fields got values for all imported assets. 
@snipe, you closed too early. Why not let me close it. At least in my case the solution was different to what @EarlRamirez wrote.
